packages/web/components/LandingPage/Resources/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Slider from 'react-slick';
+
+import Item from './Item';
+
+import theme from '../../../styles/theme';
+
+import { Wrapper, Container } from './styles';
+
+const items = [
+	{
+		title: (
+			<span>
+				Banco de
+				<br />
+				Editais
+			</span>
+		),
+		image: {
+			src: '/revenue-rafiki.svg',
+			alt:
+				'Homem de camisa laranja e gravata comemorando com as mãos para o alto e uma seta para cima ao fundo',
+		},
+		description:
+			'Encontre aqui os editais mais atuais de incentivo à pesquisa e inovação. Quem sabe você encontra aquele investimento que precisa, hein?',
+		link: {
+			href: '/announcements',
+			label: 'Ver  editais',
+		},
+	},
+	{
+		title: (
+			<span>
+				Banco de
+				<br />
+				Ideias
+			</span>
+		),
+		image: {
+			src: '/brainstorming-rafiki.svg',
+			alt:
+				'Pessoas fazendo um brainstorming com uma ilustração de uma grande lâmpada acima delas',
+		},
+		description:
+			'Procurou e não encontrou a tecnologia que você precisa? Não se preocupe. Sugirá ideias para os pesquisadores desenvolverem novas tecnologias.',
+		link: {
+			href: '/ideas',
+			label: 'Ver  ideias',
+		},
+	},
+	{
+		title: (
+			<span>
+				Banco de
+				<br />
+				pesquisadores
+			</span>
+		),
+		image: {
+			src: '/people-search-rafiki.svg',
+			alt: 'Homem segurando uma lupa gigante enquanto procura uma pessoa numa página gigante',
+		},
+		description:
+			'Lista atualizada de pesquisadores por área de atuação e suas invenções e necessidades de investimento. ',
+		link: {
+			href: '/researchers',
+			label: 'Ver  pesquisadores',
+		},
+	},
+];
+
+const responsive = [
+	{
+		breakpoint: theme.screens.large,
+		settings: {
+			slidesToShow: 2.06,
+			slidesToScroll: 2,
+		},
+	},
+	{
+		breakpoint: theme.screens.medium,
+		settings: {
+			slidesToShow: 1.06,
+			slidesToScroll: 1,
+		},
+	},
+];
+
+const Resources = () => {
+	return (
+		<Wrapper>
+			<Container>
+				<Slider
+					infinite
+					speed={500}
+					slidesToShow={3}
+					slidesToScroll={3}
+					arrows={false}
```

Comment:
This slide effect looks great on mobile screens :100:. 

However, I think we might show an indicator that this is a slide function, because it does not seem clear to the user. We can show the arrows. If we need to customize the style, we don't need to make something too fancy...just an arrow to show the user he is able to swipe right and left 👍

Reply:
@lcnogueira to make it clear that this is a slider, we show a bit of the next slide:
![image](https://user-images.githubusercontent.com/10481028/102358317-285cac00-3f8e-11eb-8bb9-ca5fdcd3deb9.png)

But just to make it clearer that it's a slider and to keep the layout cleaner, I've opted to add slider dots instead of arrows:
![image](https://user-images.githubusercontent.com/10481028/102359106-1fb8a580-3f8f-11eb-8577-c34d529e7eb7.png)

What do you think?

Reply:
Much better now! 🤝 :100: